projects/step-core/src/lib/modules/basics/injectables/statuses-colors.token.ts

```diff
@@ -5,7 +5,7 @@ import { DOCUMENT } from '@angular/common';
 type E = Pick<Required<Execution>, 'status' | 'result'>;
 type Status = E['status'] | E['result'] | 'UNKNOW' | 'DELAYED_UPDATE' | 'EMPTY';
 
-const COLORS: Record<Status, string> = {
+export const COLORS: Record<Status, string> = {
```

Comment:
You don't need to export `COLORS` constant.
Use injection token `STATUS_COLORS` instead.
like:
```
private _colors = inject(STATUS_COLORS);

readonly HEATMAP_STATUS_COLORS: Record<HeatmapStatus, string> = {
    VETOED: this._colors.TECHNICAL_ERROR,
    TECHNICAL_ERROR: this._colors.TECHNICAL_ERROR,
    ...
  };
```

We need to use the injection token only, to be sure that colors were added to document styles.